src/main/java/net/awairo/minecraft/spawnchecker/api/Renderer.java

```diff
@@ -83,24 +84,31 @@ default void addVertex(double x, double y, double z) {
             .endVertex();
     }
 
-    default void addVertex(double x, double y, double z, double u, double v) {
+    default void addVertex(double x, double y, double z, float u, float v) {
         buffer()
             .pos(x, y, z)
             .tex(u, v)
             .endVertex();
     }
 
+    default void addVertex(Matrix4f m, double x, double y, double z, float u, float v) {
+        buffer()
+            .pos(m, (float)x, (float)y, (float)z)
+            .tex(u, v)
+            .endVertex();
+    }
+
     default void addVertex(double x, double y, double z, Color color) {
         buffer()
             .pos(x, y, z)
             .color(color.red(), color.green(), color.blue(), color.alpha())
             .endVertex();
     }
 
-    default void addVertex(double x, double y, double z, double u, double v, Color color) {
+    default void addVertex(double x, double y, double z, float u, float v, Color color) {
         buffer()
             .pos(x, y, z)
-            .tex(u, v)
+            .tex((float)u, (float)v)
```

Comment:
```suggestion
            .tex(u, v)
```
引数の型 `float` に直してあるので冗長なキャストかも？